v0/destinations/google_adwords_remarketing_lists/util.js

```diff
@@ -114,14 +114,15 @@ const gaAudienceProxyRequest = async request => {
     headers,
     method,
     jobId,
-    body
+    request.body.JSON
```

Comment:
Let's use Object destructuring at top to get this much safer

Reply:
refactored